build/CodeCoverage.runsettings

```diff
@@ -1,40 +1,44 @@
-<?xml version="1.0" encoding="utf-8"?>  
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I am not sure I trust the numbers. Do the numbers you find locally match the numbers reported by the build?

Reply:
Did a local check with the latest config and the numbers are close
![{36AB0DAD-E24A-44B2-9B3E-0FB10EAEBD3D}](https://github.com/user-attachments/assets/e45f3016-e151-469b-8875-1b497093f921)

![image](https://github.com/user-attachments/assets/759cd701-be42-448a-8508-a1d28faffabb)